cmake/Modules/FindComputeCpp.cmake

```diff
@@ -242,7 +256,11 @@ function(__build_spir targetName sourceFile binaryDir fileCounter)
   add_dependencies(${targetName} ${headerTargetName})
 
   # Force inclusion of the integration header for the host compiler
-  set(compileFlags -include ${outputSyclFile})
+  if(MSVC)
+    set(compileFlags /FI ${outputSyclFile})
```

Comment:
Since this is only used to propagate the force include, maybe now would be a good time to rename it to forceIncludeFlag or similar? Since that is its only purpose.

Reply:
ok, will change.